bokeh/server/views/metadata_handler.py

```diff
@@ -0,0 +1,37 @@
+''' Provide a request handler that returns a json
+    with metadata information from the application
+
+'''
+from __future__ import absolute_import, print_function
+
+import json
+import logging
+log = logging.getLogger(__name__)
+
+from tornado import gen
+
+from .session_handler import SessionHandler
+
+class MetadataHandler(SessionHandler):
+    ''' Implements a custom Tornado handler for document display page
+
+    '''
+    def __init__(self, tornado_app, *args, **kw):
+        super(MetadataHandler, self).__init__(tornado_app, *args, **kw)
+
+    def initialize(self, *args, **kw):
+        pass
```

Comment:
It doesn't look like it can be deleted.  The application_context is passed to it in some cases so it barfs if we don't have the **kw.

I went ahead and moved it to SessionHandler and deleted it from the other places.